lib/api/api_auth_webauthn.go

```diff
@@ -77,17 +81,7 @@ type webauthnService struct {
 	credentialsPendingRegistration []config.WebauthnCredential
 	deviceName                     string
 	timeNow                        func() time.Time // can be overridden for testing
-}
-
-// State that changes often but also is not security-critical,
-// and therefore can be wiped and overwritten without much consequence if needed.
-type WebauthnVolatileState struct {
-	Credentials map[string]WebauthnCredentialVolatileState `json:"credentials,omitempty"` // Keys are base64.RawURLEncoding.EncodeToString(credential ID)
-}
-
-type WebauthnCredentialVolatileState struct {
-	SignCount   uint32    `json:"signCount,omitempty"`
-	LastUseTime time.Time `json:"lastUseTime,omitempty"`
+	volStateMut                    sync.RWMutex
```

Comment:
While it's probably guessable what makes up volatile state in this struct, imo it's still nice for the reader to be explicit: As in if a mut isn't covering everything, using an empty line to group all the fields and the mutex protecting them together, separating them from the rest (plus a comment pointing that out).

Reply:
Oh nice, I think I was wrong on the "it's probably guassable what makes up volatile state in this struct" above. I am not updating the above despite not having sent the comment - it nicely illustrates why some hint here in the form of a comment and/or grouping is likely warranted:  

The lock is applied on operations on miscDB - my guess would have been it's for the maps maps (e.g. `registrationStates`). So if this lock is specifically for `miscDB` usage, then naming it `miscDBMut` would imo be self-documenting enough. If there's something else this mut does, then a comment and/or grouping is imo warranted to hint at what that is.

Reply:
Great suggestion, thanks! Fixed in 76da6b3ac8e664c47a68ed788a019f72c19a5a4b and fdb5cf157e15b76587715a768771d02eeb2a3511.